### PR TITLE
Include security options in Requests library

### PR DESCRIPTION
**Pull request by jfryman**
Adds SSL support and removes annoying insecure platform warnings.

### CODE REVIEW

**Comment by Kami** [comment]
Requirements are managed a bit differently and fixed-requirements.txt needs to be updated (requirements.txt is generated during make step). I can take care of those changes, if needed.

File: requirements.txt [hunk 1]
```diff
@@ -26,7 +26,7 @@ python-gnupg<0.4,>=0.3.7
 python-json-logger
 python-nmap<0.4,>=0.3.4
 pyyaml<4.0,>=3.11
-requests<3.0,>=2.7.0
+requests<3.0,>=2.7.0 [security]
```
Comment (Kami) [inline requirements.txt]: Does this notation works (I always assumed it requires another entry)? :)

**Comment by jfryman** [comment]
ok. one sec.

File: requirements.txt (hunk 1 above)
Comment (jfryman) [inline requirements.txt]: https://pip.readthedocs.org/en/1.1/requirements.html#the-requirements-file-format

**Comment by Kami** [comment]
LGTM, +1

Lets merge this for now. We'll fix fixate-requirements.py and related code shortly.

File: requirements.txt (hunk 1 above)
Comment (dennybaa) [inline requirements.txt]: Thanks for the link. However i've started to dig into. 
Opposed to what is written there pip 7.1.2 **can not** parse `requests<3.0,>=2.7.0 [security]`
/cc @jfryman @Kami 
I'm looking to the issue.

Comment (dennybaa) [inline requirements.txt]: This is the right link:
https://pip.readthedocs.org/en/stable/reference/pip_install/#requirements-file-format

Should be `requests[security]<3.0,>=2.7.0` :)

**Pull request merged**